python/tk_houdini/ui_generation.py

```diff
@@ -134,6 +134,98 @@ def _itemNode(self, parent, label, id):
         node.text = id
         return item
 
+class AppCommandsPanels(AppCommandsUI):
+
+    def __init__(self, engine, commands=None, panel_commands=None):
+        self._panel_commands = panel_commands
+        super(AppCommandsPanels, self).__init__(engine, commands)
+
+        if self._panel_commands is None:
+            self._panel_commands = get_registered_panels(engine)
+
+    def create_panels(self, panels_file):
+        """Create the registered panels."""        
+
+        import hou
+
+        root = ET.Element("pythonPanelDocument")
```

Comment:
Do you think it would help with a comment with a sample XML structure?
Maybe it's overkill? Sometimes i find providing samples like that helps readability.